MainModule/Client/UI/Modern/DISABLED_Notif.lua

```diff
@@ -1,4 +1,5 @@
----@diagnostic disable: undefined-global
+client = nil
+service = nil
```

Comment:
Are you sure it's better to set the globals to nil instead of using the selene flag?

Reply:
Probably, the Selene flag only silents it within Selene, but it was still showing warnings on parsers and such. Considering the fact this is just coming from setfenv, there's no VArgs table or anything, I don't see a problem with this.

Reply:
Ok thats understandable